meshtastic/node_runtime/content_runtime.py

```
@@ -0,0 +1,536 @@
+        try:
+            has_ringtone_response = callable(
+                has_field
+            ) and has_field(  # pylint: disable=not-callable
+                "get_ringtone_response"
+            )
+        except (TypeError, ValueError):
+            has_ringtone_response = False
+        if not has_ringtone_response:
+            logger.warning("Unexpected ringtone response without raw ringtone data")
+            return (True, None)
+        else:
```

Ruff (0.15.7)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)